packages/circle-react-elements/src/lib/constants.ts

```diff
@@ -0,0 +1,86 @@
+import { Blockchain as LibBlockchain } from './types';
+
+export const TestnetBlockchain = {
+  ArbSepolia: 'ARB-SEPOLIA',
+  AvaxFuji: 'AVAX-FUJI',
+  EthSepolia: 'ETH-SEPOLIA',
+  MaticAmoy: 'MATIC-AMOY',
+  SolDevnet: 'SOL-DEVNET',
+  UniSepolia: 'UNI-SEPOLIA',
+} as const;
+
+export const Blockchain = {
+  Arb: 'ARB',
+  Avax: 'AVAX',
+  Eth: 'ETH',
+  Evm: 'EVM',
+  Matic: 'MATIC',
+  Near: 'NEAR',
+  Sol: 'SOL',
+  ArbSepolia: 'ARB-SEPOLIA',
+  AvaxFuji: 'AVAX-FUJI',
+  EthSepolia: 'ETH-SEPOLIA',
+  EvmTestnet: 'EVM-TESTNET',
+  MaticAmoy: 'MATIC-AMOY',
+  NearTestnet: 'NEAR-TESTNET',
+  SolDevnet: 'SOL-DEVNET',
+  UniSepolia: 'UNI-SEPOLIA',
+} as const;
+
+export const BlockchainLabelMap: Record<LibBlockchain, string> = {
+  [Blockchain.ArbSepolia]: 'Arbitrum Sepolia',
```

Comment:
if the values/enums are not offered in the Circle SDK, we can still just use strings that match the SDK `Blockchain` type, like

```
const BlockchainLabelMap: Record<Blockchain, string> = {
'ARB-SEPOLIA': 'Arbitrum Sepolia'
// etc.
}
```
I'd suggest to depend as much on the SDK types rather than introduce our own

Reply:
fixed